hedgehog-classes.cabal

```diff
@@ -151,7 +151,7 @@ library
     , base >= 4.12 && < 4.18
     , binary >= 0.8 && < 0.9
     , containers >= 0.5 && < 0.7
-    , hedgehog >= 1 && < 1.2
```

Comment:
I'm afraid I don't have time to add CPP back compat for < 1.2. I suggest if people need this, then they can just use an older release of hedgehog-classes, or maybe someone else can extend this PR.